test/plugin/constants.rb

```diff
@@ -140,6 +140,11 @@ module Constants
     detect_subservice false
   )
 
+  PROMETHEUS_ENABLE_CONFIG = %(
+    enable_monitoring true
```

Comment:
The code above suggests the default for `enable_monitoring` is `false` but it's true here - should they be consistent?